protokolle/29-04-2020_protokoll.md

```diff
@@ -0,0 +1,262 @@
+# 2020-04-29 14:00 - 15:30 Wetterdatenaufbereitung mit PostgreSQL
+
+## Feedback zur Übung / Generell:
+
+    * Relativ wenig Antworten  
+
+    * Werden die nächsten Tage bewertet  
+
+    
+
+## Fragen zu git und GitHub:
+    
+    *Wie kann ich einen Pull Request über git scm in den Master starten?* Hab das jetzt über GitHub gemacht. [Link](https://github.com/htw-pv3/literature/pull/9) 
+	
+    ->Die Pull Requests (PR) sollten über GitHub erstellt werden. Das geht auch über die GitBash, aber Ziel des PR ist eine Dokumentation und Kommunikation (Review mit 4-Augen-Prinzip)  
+    
+	
+    *Wie sieht der "git workflow" genau aus?*
+	
+    -> Habe dazu ein kurzes Video-Tutorial gemacht.  
+    
+	
+    *Was ist eine "gute Commit-Message"?*
+	
+    -> Verb - Nomen - #Ticketnummer
+	
+	
+    *Was ist eine "schlechte Commit-Message"?*
+    -> 
+    
+	
+    *Darf ich jemals direkt auf den Master-Branch pushen?*
+	
+    -> Nein!
+
```

Comment:
- to much indentation
- add space after *

Reply:
my goal was to display the texts in cursive with * 
Oh well :D